sormas-backend/src/main/java/de/symeda/sormas/backend/common/StartupShutdownService.java

```diff
@@ -546,8 +546,13 @@ private void createOrUpdateDefaultUser(Set<UserRole> userRoles, String username,
 
 		User existingUser = userService.getByUserName(username);
 
+		boolean allUserRolesExist = userRoles != null && !userRoles.isEmpty() && !userRoles.stream().anyMatch(userRole -> userRole == null);
```

Comment:
Use `collections4.CollectionUtils.isEmpty()` instead of `userRoles != null && !userRoles.isEmpty()`.